### PR TITLE
Suggest scienta/doctrine-json-functions for complex JSON queries.

### PR DESCRIPTION
**Pull request by bpolaszek**
I think both packages are related and deserve a handshake.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -26,7 +26,8 @@
     "symfony/phpunit-bridge": "^3.3 || ^4.0"
   },
   "suggest": {
-    "symfony/framework-bundle": "To use the provided bundle."
+    "symfony/framework-bundle": "To use the provided bundle.",
+    "scienta/doctrine-json-functions": "To run JSON functions inside query builders."
```
Comment (dunglas) [inline composer.json]:
```suggestion
    "scienta/doctrine-json-functions": "To add support for JSON functions in DQL."
```

File: README.md [hunk 2]
```diff
@@ -231,6 +228,14 @@ Then, you need to set an option of in the column mapping:
 
 Yes.
 
+**Can I use native [MySQL](https://dev.mysql.com/doc/refman/en/json.html) / [PostgreSQL](http://www.postgresql.org/docs/current/static/datatype-json.html) JSON functions?**
+
```
Comment (dunglas) [inline README.md]:
```suggestion
```

File: README.md [hunk 3]
```diff
@@ -231,6 +228,14 @@ Then, you need to set an option of in the column mapping:
 
 Yes.
 
+**Can I use native [MySQL](https://dev.mysql.com/doc/refman/en/json.html) / [PostgreSQL](http://www.postgresql.org/docs/current/static/datatype-json.html) JSON functions?**
```
Comment (dunglas) [inline README.md]:
```suggestion
**Can I use the native [PostgreSQL](http://www.postgresql.org/docs/current/static/datatype-json.html) and [MySQL](https://dev.mysql.com/doc/refman/en/json.html) /JSON functions?**
```

File: README.md [hunk 4]
```diff
@@ -231,6 +228,14 @@ Then, you need to set an option of in the column mapping:
...
+
+
+You can execute complex queries using [native queries](https://www.doctrine-project.org/projects/doctrine-orm/en/current/reference/native-sql.html). 
+
+To run JSON functions inside query builders or DQL statements, have a look at [scienta/doctrine-json-functions](https://github.com/ScientaNL/DoctrineJsonFunctions)
```
Comment (dunglas) [inline README.md]:
```suggestion
Alternatively, install [scienta/doctrine-json-functions](https://github.com/ScientaNL/DoctrineJsonFunctions) to be able to use run JSON functions in DQL and query builders.
```

File: README.md [hunk 5]
```diff
@@ -231,6 +228,14 @@ Then, you need to set an option of in the column mapping:
 
 Yes.
 
+**Can I use native [MySQL](https://dev.mysql.com/doc/refman/en/json.html) / [PostgreSQL](http://www.postgresql.org/docs/current/static/datatype-json.html) JSON functions?**
+
+
+You can execute complex queries using [native queries](https://www.doctrine-project.org/projects/doctrine-orm/en/current/reference/native-sql.html).
```
Comment (dunglas) [inline README.md]:
```suggestion
Yes! You can execute complex queries using [native queries](https://www.doctrine-project.org/projects/doctrine-orm/en/current/reference/native-sql.html). 
```

**Comment by bpolaszek** [comment]
> Good idea! It could also be nice to reference this library on https://github.com/ScientaNL/DoctrineJsonFunctions.

Hold my beer.

**Comment by bpolaszek** [comment]
FYI https://github.com/ScientaNL/DoctrineJsonFunctions/pull/44 has been merged :simple_smile:

**Comment by Toflar** [comment]
Thanks @bpolaszek 😊

**Pull request merged**